misc/scripts/build.sh

```diff
@@ -140,6 +139,9 @@ else
     -e "ESX=$2" \
     -e "VM1=$3" \
     -e "VM2=$4" \
+    -e "DS1=$DS1" \
+    -e "VM1NAME=$VM1NAME" \
```

Comment:
I'm using the current storage-main. I read govc is tested up to 6.5 ESX. I plan on using govc once this change is merged.